src/feature/RansacFeatureSetMatcher.cpp

```diff
@@ -123,5 +123,8 @@ double RansacFeatureSetMatcher::matchSets(const std::vector<InterestPoint *> &re
 	pointCorrespondences[i] = std::make_pair(correspondences[i].first->getPosition(), correspondences[i].second->getPosition());
     }
     compute2DPose(pointCorrespondences, transformation);
-    return verifyHypothesis(reference, data, transformation, correspondences);
+    double score = verifyHypothesis(reference, data, transformation, correspondences);
```

Comment:
For reference, this one is implemented here: https://github.com/clearpathrobotics/flirtlib/blob/master/src/feature/AbstractFeatureSetMatcher.cpp#L44